Update Developer List

Hitakashi opened the pull request:
Taken directly from the subreddits CSS. https://www.reddit.com/r/Guildwars2/wiki/config/stylesheet

Remove:
 - darapotocska

Add
- ANET_Byfield
- Anet_Evangelos
- Anet-PaulS
- JessicaLPrice
- Anet_Amy
- GM_Perfbot
- ANet-Katelyn
- JeffreyVaughn_ANet
- anet-daheeim
- Anet_Greg

windwarrior commented:
Thank you very much, I did a small cleanup of the date variable names in the contexts not very fan of "date2" ;).

Hitakashi committed:
Update devs.json

Remove:
 - darapotocska
Add
- ANET_Byfield
- Anet_Evangelos
- Anet-PaulS
- JessicaLPrice
- Anet_Amy
- GM_Perfbot
- ANet-Katelyn
- JeffreyVaughn_ANet
- anet-daheeim
- Anet_Greg

windwarrior commented:
Hai! Thank you very much for updating the list of devs! I haven’t played the game in a while so I didn’t keep up…

I’ll accept this over the weekend because I need to also ‘release’ it to GitHub pages for which I need a proper non-mobile device. If I don’t do it Saturday, could you just remind me Sunday?

windwarrior commented:
I find it interesting that Jessica Price is still marked a dev in the subreddit css, but yeah, just taking the list from the sub verbatim... Thank you very much
